### PR TITLE
 PLAT-115303: Sync up with sandstone/Scroller

### PR DESCRIPTION
**Pull request by jeonghee27**
This PR syncs up agate/Scroller with sandstone/Scroller .
Some of bug fixes from sandstone/Scroller needs to be synced to agate/Scroller .
Please refer to the ticket for details.

Enact-DCO-1.0-Signed-off-by: Jeonghee Ahn <jeonghee27.ahn@lge.com>

### CODE REVIEW

File: samples/sampler/stories/default/Scroller.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+import BodyText from '@enact/agate/BodyText';
+import Scroller from '@enact/agate/Scroller';
+
+const prop = {
+	direction: ['both', 'horizontal', 'vertical'],
+	focusableScrollbarOption: {
+		false: false,
+		true: true,
+		'&quot;byEnter&quot;': 'byEnter'
```
Comment (MikyungKim) [inline samples/sampler/stories/default/Scroller.js:17]: I don't think we have `byEnter` in agate.

File: samples/sampler/stories/default/Scroller.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import {action} from '@enact/storybook-utils/addons/actions';
```
Comment (MikyungKim) [inline samples/sampler/stories/default/Scroller.js:1]: Would you please order this?

File: samples/sampler/stories/default/Scroller.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+};
+
+const ScrollerConfig = mergeComponentMetadata('Scroller', UiScrollerBasic, Scroller);
+
+storiesOf('Agate', module)
+	.add(
+		'Scroller',
+		() => (
+			<Scroller
+				style={{height: ri.scaleToRem(540), width: ri.scaleToRem(1540)}}
```
Comment (MikyungKim) [inline samples/sampler/stories/default/Scroller.js:30]: Why it has a fixed size?

Comment (MikyungKim) [reply samples/sampler/stories/default/Scroller.js:30]: Let's remove `:not([style])` from `ThemeEnvironment.module.less` and remove styles from Scroller and VL.

File: Scroller/useThemeScroller.js [hunk 4]
```diff
@@ -248,7 +248,6 @@ const useThemeScroller = (props) => {
 	delete rest.scrollAndFocusScrollbarButton;
 	delete rest.scrollContainerContainsDangerously;
 	delete rest.scrollContainerHandle;
-	delete rest.scrollContainerRef;
 	delete rest.scrollContentHandle;
```
Comment (0x64) [inline Scroller/useThemeScroller.js:251]: Rather than deleting `scrollContentHandle` from `rest`, what about destructuring it from `props` directly?

**Pull request merged**